<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1200</width>
      <height>780</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QPushButton" name="qna_Btn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>QnA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="test_Btn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>문제 풀기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>600</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>점수 및 등급</string>
      </property>
     </widget>
     <widget class="QPushButton" name="learning_Btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>학습하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chatpage_Btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>440</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>상담</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>1200</width>
        <height>780</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>image/boack.png</pixmap>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>qna_Btn</zorder>
     <zorder>test_Btn</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>learning_Btn</zorder>
     <zorder>chatpage_Btn</zorder>
    </widget>
    <widget class="QWidget" name="learning_page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>1021</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="info_textBrowser">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="back_Btn1">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>690</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(80, 188, 223, 0.5)</string>
      </property>
      <property name="text">
       <string>처음으로</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>550</y>
        <width>1021</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(80, 188, 223, 0.5)rgb(255, 255, 255)</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="answer_textBrowser">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>620</y>
        <width>1021</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;학습하기&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>search_lineEdit</zorder>
     <zorder>answer_textBrowser</zorder>
     <zorder>back_Btn1</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QWidget" name="test_page">
     <widget class="QPushButton" name="back_Btn2">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>710</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>처음으로</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>16</width>
        <height>765</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>5</y>
        <width>1131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>1155</x>
        <y>10</y>
        <width>20</width>
        <height>765</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>770</y>
        <width>1131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>1121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>309</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;문제는 총 25문제이며, 60점 이상 합격입니다.&lt;/p&gt;&lt;p&gt;제한 시간은 10분입니다.&lt;/p&gt;&lt;p&gt;정답이 여러 개 있을 수 있습니다.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>16</width>
        <height>110</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>20</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;출제 참고 자료(공공데이터포털)&lt;/p&gt;&lt;p&gt;https://www.data.go.kr/data/15100163/fileData.do&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>1081</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout1">
       <item>
        <widget class="QLabel" name="testLabel1">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>1081</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout2">
       <item>
        <widget class="QLabel" name="testLabel2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>450</y>
        <width>1081</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout3">
       <item>
        <widget class="QLabel" name="testLabel3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>610</y>
        <width>1081</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="testLayout4">
       <item>
        <widget class="QLabel" name="testLabel4">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="next_Btn">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>710</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>제출 후 다음으로</string>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>10</y>
        <width>16</width>
        <height>110</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="count_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;제출 한 문제 수 : 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="time_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;경과 시간 : 0분 0초&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_8">
      <property name="geometry">
       <rect>
        <x>565</x>
        <y>10</y>
        <width>16</width>
        <height>110</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="result_Btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>710</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>결과 확인하기</string>
      </property>
     </widget>
     <zorder>line_2</zorder>
     <zorder>line_4</zorder>
     <zorder>line_5</zorder>
     <zorder>line_3</zorder>
     <zorder>line</zorder>
     <zorder>verticalLayoutWidget_2</zorder>
     <zorder>line_6</zorder>
     <zorder>verticalLayoutWidget_3</zorder>
     <zorder>verticalLayoutWidget_4</zorder>
     <zorder>verticalLayoutWidget_5</zorder>
     <zorder>verticalLayoutWidget_6</zorder>
     <zorder>verticalLayoutWidget_7</zorder>
     <zorder>line_7</zorder>
     <zorder>verticalLayoutWidget_8</zorder>
     <zorder>verticalLayoutWidget_9</zorder>
     <zorder>line_8</zorder>
     <zorder>result_Btn</zorder>
     <zorder>next_Btn</zorder>
     <zorder>back_Btn2</zorder>
    </widget>
    <widget class="QWidget" name="result_page">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>700</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>돌아가기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;결과 확인&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>1181</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:72pt; font-weight:600;&quot;&gt;내 점수&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>1181</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;채점 결과&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="result_textBrowser"/>
       </item>
      </layout>
     </widget>
     <zorder>label_2</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>verticalLayoutWidget_10</zorder>
    </widget>
    <widget class="QWidget" name="chat_Page">
     <widget class="QTextBrowser" name="chat_textBrowser">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>861</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="chat_lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>590</y>
        <width>741</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>680</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>돌아가기</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="qna_Page">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>qna page</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>1101</width>
        <height>561</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>100</number>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="newQnA_Btn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>700</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>QnA 등록하기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>700</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
